workers/unity/Assets/Gdk/TestUtils/Tests/Editmode/HybridGdkSystemTests.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using JetBrains.Annotations;
+using NUnit.Framework;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Improbable.Gdk.TestUtils.EditmodeTests
+{
+    public class HybridSystemTests
+    {
+        private struct TestPreparation : IComponentData
+        {
+            public int Value;
+        }
+
+        [UsedImplicitly]
+        [DisableAutoCreation]
+        private class ExampleHybridSystem : ComponentSystem
+        {
+            private struct PreparationData
+            {
+                public int Length;
+
+                // The ComponentArray<> here makes this a hybrid system.
+                public ComponentArray<Rigidbody> GameObjectRigidBody;
```

Comment:
Will this not always be unity "null" in these tests?

Reply:
For these tests yes, but I just want to test that "any `ComponentArray<>` would fail"

Reply:
Adding a comment to explain it better